.github/workflows/test-backend.yaml

```diff
@@ -17,10 +17,6 @@ jobs:
       - uses: actions/checkout@v4
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
-      - name: start backend
-        uses: ./.github/actions/run-backend
-        with:
-          django_secret_key: ${{ env.DJANGO_SECRET_KEY }}
```

Comment:
Happy we can optimize this. I guess we can remove `DJANGO_SECRET_KEY: ${{ secrets.DJANGO_SECRET_KEY }}` in the env section near the top of this file.